rd_ui/app/scripts/ng_highchart.js

```diff
@@ -325,6 +325,36 @@
                 series = seriesCopy;
               }
 
+              if (chartOptions['sortY'] === true || chartOptions['sortY'] === undefined) {
+                var sortable = true;
+
+                for (var i = 0; i < series.length; i++) {
+                  if (series[i].data.length != 1) {
```

Comment:
Probably not...

But, now that I better understand what you did here, I think that we can keep this logic enabled for everyone and we don't need the toggle. 

If you think we should keep the toggle, then we need to find different wording for it as "Sort Y values" is confusing (or implement proper sorting of Y values for all scenarios...).

Also the sorting code becomes:
```javascript
series = _.sortBy(series, function(s) { return s.data[0].y });
```

And detection:
```javascript
var sortable = _.every(series, function(s) { returns s.data.length == 1 });
```